### ISSUE TITLE
Use of web based css File.

### ISSUE THREAD

**Issue by Jebik** [fluentdesk/fluent-themes#55]
I'm currently updating security of my website... And one of the security setting is to transmit header informations to tell to web browser to only use my content. 

Most of the theme use a link to https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css. 

Wouldn't it be better to add the css to your theme and then link it in generated folder?
For keeping the html version offline readable?

**Comment by hacksalot** [comment]
Yes. At least that option should be provided. Let's look into this.

**Comment by hacksalot** [comment]
This would be similar to the ability to switch between embedded (via `<style>`) versus linked (via `<link>`) CSS styles, which we support because it comes in handy when users are doing a lot of local-only HTML viewing and PDF generation.
